libs/ledger/include/ledger/chaincode/contract.hpp

```diff
@@ -291,5 +293,23 @@ StateAdapter::Status Contract::SetStateRecord(T const &record, ConstByteArray co
   return state().Write(std::string{key}, data.pointer(), data.size());
 }
 
+class ContractAttachHelper
```

Comment:
NB: `Helper` is not very descriptive here. Even `Attacher` would probably be more readable.

Reply:
Done.